bazel/repositories.bzl

```diff
@@ -16,11 +16,18 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 
 def proxy_wasm_rust_sdk_repositories():
+    maybe(
+        http_archive,
+        name = "rules_cc",
```

Comment:
Why is this needed? I didn't see use of a cc build rule added anywhere.

Reply:
It's not added anywhere in this PR, but there is an existing reference in the auto-generated `bazel/cargo/remote/alias_rules.bzl` to `@rules_cc`.

I think `rules_rust` got more picky about it, because it fails without adding it, but I didn't investigate further:
```
$ bazelisk --noworkspace_rc run --noenable_bzlmod //bazel/cargo:crates_vendor
```
```
ERROR: Failed to load Starlark extension '@@rules_cc//cc:action_names.bzl'.
Cycle in the workspace file detected. This indicates that a repository is used prior to being defined.
The following chain of repository dependencies lead to the missing definition.
 - @@rules_cc
This could either mean you have to add the '@@rules_cc' repository with a statement like `http_archive` in your WORKSPACE file (note that transitive dependencies are not added automatically), or move an existing definition earlier in your WORKSPACE file.
ERROR: Error computing the main repository mapping: cycles detected during computation of main repo mapping
```

Reply:
I see, SGTM.